pages/learn/manage/account.md

```diff
@@ -60,15 +60,16 @@ When an application needs to be shared with more than one user, the application
 
 ### Member types
 
-| Member Type | Add members   | Delete App  | Add/Remove device  | Device actions  | Tags | Dev Env Variables | SSH access | Push | Fleet Env Variables | Fleet actions |
+| Member Type   | Add members   | Delete App  | Add/Remove device  | Device actions  | Tags | Dev Env Variables | SSH access | Push | Fleet Env Variables | Fleet actions |
 | ------------- |:-------------:| ----:| ----:| ----:| ----:| ----:| ----:| ----:| ----:| ----:|
-| Developer  | No       | No   | Yes   | Yes  | Yes  | Yes | Yes | Yes | Yes | Yes |
-| Operator   | No       | No   | Yes   | Yes  | Yes  | Yes | Yes | No  | No  | No  |
-| Observer   | No       | No   | No   | No   | No   | No  | No  | No  | No  | No  |
+| Administrator | Yes      | Yes  | Yes  | Yes  | Yes  | Yes | Yes | Yes | Yes | Yes |
+| Developer     | No       | No   | Yes  | Yes  | Yes  | Yes | Yes | Yes | Yes | Yes |
+| Operator      | No       | No   | Yes  | Yes  | Yes  | Yes | Yes | No  | No  | No  |
+| Observer      | No       | No   | No   | No   | No   | No  | No  | No  | No  | No  |
 
-#### Owner
+#### Administrator
 
-The application owner is the user who first creates an application. The owner is the only user who can add other application members or delete the application.
+A new application is always created by an an [administrator][administrator] of the organization. Administrators are the only users who can add other application members or delete the application. Learn more about the [administrator role](adminstrator) in an organization's scope.
```

Comment:
```suggestion
A new application in balenaCloud can only be created by an [administrator][administrator] of an organization. Administrators are the only users who can add other application members or delete the application. Learn more about the [administrator role](adminstrator) in an organization.
```

Remove 2 `an`
Generalized it a bit and removed the scope bit.

Reply:
Well @vipulgupta2048 the change type squash is so that this PR doesn't get merged before I have the change to merge my commits. even in case someone approves :)